examples/twitter/tweet_at_cozmo.py

```diff
@@ -218,17 +220,18 @@ def on_tweet_from_user(self, json_data, tweet_text, from_user, is_retweet):
 
         tweet_id = json_data.get('id_str')
 
-        cmd_func, cmd_args = self.extract_command_from_string(tweet_text)
-
-        reply_prefix = "@" + from_user_name + " "
-        if cmd_func is not None:
-            kw_args = {'tweet_id': tweet_id, 'reply_prefix': reply_prefix}
-            result_string = cmd_func(cmd_args, kw_args)
-            if result_string:
-                self.post_tweet(reply_prefix + result_string, tweet_id)
-        else:
-            self.post_tweet(reply_prefix + "Sorry I don't understand, available commands are: "
-                            + str(self.get_supported_commands()), tweet_id)
+        command_list = self.extract_command_from_string(tweet_text)
+        print(command_list)
+        for cmd_func, cmd_args in command_list:
+            reply_prefix = "@" + from_user_name + " "
+            if cmd_func is not None:
+                kw_args = {'tweet_id': tweet_id, 'reply_prefix': reply_prefix}
+                result_string = cmd_func(cmd_args, kw_args)
+                if result_string:
+                    self.post_tweet(reply_prefix + result_string, tweet_id)
+            else:
+                self.post_tweet(reply_prefix + "Sorry I don't understand, available commands are: "
```

Comment:
Because of the change to a for loop, there's no longer a post_tweet response if the list returned is empty.